.github/azure-devops/release.yml

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+---
+
+trigger:
+  tags:
+    include:
+      - v*
```

Comment:
Suggest updating docs to indicate how to generate a release after this change.

Reply:
The instructions are the same. Assuming this works out correctly – and I can't unfortunately guarantee that until the next release, but I will update if there's a failure – it's designed to be triggered by existing elements of the release to automatically invoke this job.